node/_fs/_fs_constants.ts

```diff
@@ -22,4 +22,18 @@ export const {
   UV_FS_COPYFILE_EXCL,
   UV_FS_COPYFILE_FICLONE,
   UV_FS_COPYFILE_FICLONE_FORCE,
+  O_RDONLY,
```

Comment:
These also need to be exported in `node/constants.ts` to pass the CI check

Reply:
fixed 👍